adapters/data_saver.py

```diff
@@ -0,0 +1,34 @@
+import datetime
+from abc import ABC
+
+from ports.data_parser import FileSaveError
+
+
+class FileWriter(ABC):
+
+    def write(self, content: str, output_path: str) -> None:
```

Comment:
Please add `@abstractmethod` decorator. This will prevent instantiating classes deriving from `FileWriter` that do not implement `write` function

Reply:
done